### DISCUSSION TITLE
docs: where can I find the default configuration?

### DISCUSSION THREAD

**Discussion by OliverNChalk** [netdata/netdata#14988] [Q&A]
I think I must be missing something or doing something wrong. Whenever I want to edit a particular config value (i.e. disk usage warn/crit thresholds) it turns into a game of battleships, where I am trying to guess the name of the default configuration file.

Is there a comprehensive and complete documentation of the default configuration files somewhere? Additionally, I assume the config files are missing from my install so that I can have my config auto-update if I do not introduce a manual configuration? If this is the case, could we write an empty file instead to indicate default, this would let users know what configs actually exist and are in play.

Right now it is extremely hard, at least to me (so maybe im missing something), to have any insight into what configuration is currently running and what options are available for tuning.

### Edit 1

Someone in discord pointed out `sudo /etc/netdata/edit-config --list` can be used to explore possible configuration options. Additionally, [This page](https://learn.netdata.cloud/docs/getting-started/manage-and-configure/configuration) provides an intro to configuration. Perhaps a call out in some of the doc pages saying this is pre-requisite reading before attempting to configure `netdata` will help guide those in a rush like myself

**Comment by ilyam8** [comment]
> Perhaps a call out in some of the doc pages saying this is pre-requisite reading before attempting to configure netdata will help guide those in a rush like myself

Hey, @OliverNChalk. I see that we have [Configuration](https://learn.netdata.cloud/docs/getting-started/install-netdata#configuration) on the [Install Netdata/Get started](https://learn.netdata.cloud/docs/getting-started/install-netdata) page. Can you please elaborate on your suggestion? On which page do we need to add that information?

**Reply by OliverNChalk** [reply]
Hey, I believe my frustration came from the same root issue that was resolved by clean installing from the apt repository. Essentially, due to netdata's somewhat complex configuration system I was assuming config was the reason for applications not showing up on my dashboard. However, it is now evident that invalid permissions was the actual reason. As such, I think my comment is not entirely relevant/fair. I revisited the various plugin pages and I think they are sufficiently approachable as documentation - in fact the configuration works fine now that I have the root cause patched.

If I think a little further, my key mistake was:

Assuming no `applications` on the dashboard meant `invalid applications configuration`, when in fact it was purely a permissioning issue. `error.log` may not be sufficiently easy/obvious to third-party users for debugging and it appears that the systemd output doesn't always complain about failing units. So I wonder if these sorts of issues could not be routed to the Netdata cloud dashboard and presented to the user.

For instance, if I had been told apps.plugin failed to start with a read error that might have clued me in that my configuration was not the source of the issue.
